templates/launchd.plist

```diff
@@ -54,5 +54,12 @@
       <key>BUILDKITE_AGENT_CONFIG</key>
       <string>/Users/your-build-user/.buildkite-agent/buildkite-agent.cfg</string>
     </dict>
+    <key>LimitLoadToSessionType</key>
+    <array>
+      <string>Aqua</string>
+      <string>LoginWindow</string>
+      <string>Background</string>
+      <string>StandardIO</string>
+    </array>
```

Comment:
I often see `<string>System</string>` as well (for instance Homebrew creates it). I'm not sure it's necessary though.